packages/app/playwright.config.ts

```diff
@@ -33,8 +33,8 @@ export default defineConfig({
     video: 'retain-on-failure',
   },
 
-  /* Global test timeout - increased from default 30s to 60s to reduce flaky test failures */
-  timeout: 60 * 1000,
+  /* Global test timeout - CI needs more time than local */
+  timeout: 45 * 1000,
```

Comment:
lower the timeout to prevent long-running tests. Can probably tune this even further in the future.